osu.Game/Screens/Play/Player.cs

```diff
@@ -790,17 +795,25 @@ private void progressToResults(bool withDelay)
             Scheduler.Add(resultsDisplayDelegate);
         }
 
-        private void beginScoreImport()
+        /// <summary>
+        /// Ends replay recording and runs <see cref="prepareAndImportScore"/> only when results can be shown
```

Comment:
This xmldoc shares nothing in common with the method name. Summaries in xmldoc should *expand* on the method name, not describe two different processes.